launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/Launcher.kt

```diff
@@ -96,11 +98,9 @@ class Launcher(val args: Array<String>) {
         }
     }
 
-    fun runBuildWrapperOnce() = workFile("gradle/wrapper/gradle-wrapper.properties") {
-        if (!exists()) {
-            println("Generating Gradle wrapper files")
-            runBuild(listOf("wrapper", "-Plauncher.wrapper=true"))
-        }
+    fun runBuildWrapperOnce() = workFileOnce("gradle/wrapper/gradle-wrapper.properties") {
+        println("Generating Gradle wrapper files")
+        runBuild(listOf(listOf("wrapper", "-Plauncher.wrapper=true"), wrapperArgs).flatten())
```

Comment:
why not just list1 + list2 :) ? kotlin has operator overloading done nicely

Reply:
Ah, cool